### ISSUE TITLE
Linear transform is not taken into account to crop with MarkupsROI

### ISSUE THREAD

**Issue by tlfaller** [Slicer/Slicer#6028]
The MarkupsROI can be used to crop a 3D rendering volume. No transform is needed to rotate it.
However, there is an unexpected behavior when a linear transform is applied to the Markups ROI. The MarkupsROI box is indeed shifted, but the cropped volume rendering is not updated accordingly (it remains unchanged at the same place). 
As a result the ROI and the cropped volume rendering regions do not correspond.

## Steps to reproduce

- Load MRHead volume from Sample Data module
- Add a MarkupsROI (from Markups module)
- Go to volume rendering module
- Select MRHead_1 as volume and the new MarkupsROI as ROI ; enable cropping and display the ROI (the volume is cropped)
- Go to the Transform module
- Create a new linear transform and apply it to the MarkupsROI
- Translate the Markups ROI

### Result: 
The Markups ROI is translated but the volume rendering is left unchanged. As a result, the volume rendering does not correspond to the ROI.
![MarkupsROI3](https://user-images.githubusercontent.com/85121214/142185773-b0336d22-a355-4519-b57f-2580a69d0fba.png)

## Expected behavior

The volume rendering should be updated so that it corresponds to the ROI location (as with AnnotationROI).

## Environment
- Slicer version: Slicer-4.11.20210226
- Operating system: Linux - Ubuntu 20.04.1 LTS

**Comment by Sunderlandkyl** [comment]
What version of Slicer are you using? I just tested it on the latest nightly and the transform is applied to the ROI cropping.

**Comment by jamesobutler** [comment]
It appears @tlfaller  has indicated they were using Slicer version: Slicer-4.11.20210226 as indicated in their original report. @tlfaller try the latest Slicer preview build and post here if that has the expected behavior for you.

Occasionally it comes up about a bug in Slicer stable that has already been fixed in the latest Slicer Preview. I wonder if it would be good to mention in the Bug Report template (https://github.com/Slicer/Slicer/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) that if the version used was a Slicer stable release that they should first check if the issue is also present in the latest Slicer Preview. That could potentially reduce the number of bug reports that have already been fixed and the usual response to bugs found in the stable release of “try the latest Slicer preview and report back”. 

Knowing if there is a bug in the Stable release, but not in the latest Slicer preview build would only be helpful when determining if a bug fix needs to be backported to a maintenance branch for a new patch release of stable, but Slicer hasn’t done such a patch release type since maybe 4.10.2 where it wasn’t just the same as latest `master` at that time.

**Comment by tlfaller** [comment]
Thank you for your quick responses. Indeed it works on a newer version of
Slicer.

Le mer. 17 nov. 2021 à 16:00, James Butler ***@***.***> a
écrit :

**Comment by lassoan** [comment]
> I wonder if it would be good to mention in the Bug Report template

Good idea. I've added a note there (https://github.com/Slicer/Slicer/commit/26111a1f901658d7463334a32e050cc38e93e1c9#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5).

**Comment by lassoan** [comment]
Closing, as the issue has been fixed in recent Slicer versions.
